src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -115,11 +132,27 @@ export default {
 		},
 		trackCategory({ success }, category) {
 			if (success) this.$kvTrackEvent('loan-card', 'add-to-basket', `${category}-lending-home`);
+		},
+		closeToast() {
+			this.$refs.welcomeToastMessage.close();
```

Comment:
It looks like there's an issue with the toast refs in this component. There's a max stack depth when this method is called. I logged `this.$refs`, and there are a ton of refs!

Reply:
Nice catch! Its fixed.